src/scss/base/validation.scss

```diff
@@ -0,0 +1,6 @@
+input {
```

Comment:
I believe we use two spaces for indentation

Reply:
@squallstar fixed